### ISSUE TITLE
 mismatched input 'inputFileListPaths' expecting 'inputPaths'

### ISSUE THREAD

**Issue by mc7h** [Karumi/Kin#36]
Encountered after adding a standard Carthage copy frameworks phase.

```
ERROR: line 294:3 mismatched input 'inputFileListPaths' expecting 'inputPaths'
ERROR: line 299:3 mismatched input 'outputFileListPaths' expecting 'outputPaths'
ERROR: line 312:3 mismatched input 'inputFileListPaths' expecting 'inputPaths'
ERROR: line 317:3 mismatched input 'outputFileListPaths' expecting 'outputPaths'
ERROR: line 330:3 mismatched input 'inputFileListPaths' expecting 'inputPaths'
ERROR: line 337:3 mismatched input 'outputFileListPaths' expecting 'outputPaths'
```

Lines: 325-346 of my .pbxproj

```
		A7CD39BE222E3AF6003FA014 /* Run Copy Carthage Frameworks */ = {
			isa = PBXShellScriptBuildPhase;
			buildActionMask = 2147483647;
			files = (
			);
			inputFileListPaths = (
			);
			inputPaths = (
				"$(SRCROOT)/Carthage/Build/iOS/Alamofire.framework",
				"$(SRCROOT)/Carthage/Build/iOS/AlamofireImage.framework",
			);
			name = "Run Copy Carthage Frameworks";
			outputFileListPaths = (
			);
			outputPaths = (
				"$(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/Alamofire.framework",
				"$(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/AlamofireImage.framework",
			);
			runOnlyForDeploymentPostprocessing = 0;
			shellPath = /bin/sh;
			shellScript = "/usr/local/bin/carthage copy-frameworks\n";
		};
```

**Comment by pedrovgs** [comment]
Hi @mc7h we'd love to review this issue and I was wondering if you could upload a  .pbxproj file reproducing the error we can use for testing purposes.

**Comment by mc7h** [comment]
Here you go @pedrovgs 
[mc7h-project.pbxproj.txt](https://github.com/Karumi/Kin/files/2938381/mc7h-project.pbxproj.txt)

**Comment by icedice** [comment]
I'm seeing this as well on a perfectly good project...

**Comment by Serchinastico** [comment]
It's due to an outdated grammar, you can safely ignore them. In the meantime I'm preparing a new release with an update to include these new syntax elements inside Kin. I'll keep you posted.

**Comment by Serchinastico** [comment]
Released a fix in Kin 1.7.2!
